CSharp/BatchExplorer/README.md

```diff
@@ -1,5 +1,7 @@
 ## Azure Batch Explorer
 
+**Note:** For an officially supported cross-platform client tool to browse Azure Batch resources see [Batch Labs](https://github.com/Azure/BatchLabs). Batch Explorer will continue to exist in this repository only as a sample and may not be updated to expose the latest features of Batch.
```

Comment:
I would like to be a little less passive here.  

In my mind, BE is now in sustained engineering, will not be getting all new features and we are no longer accepting PRs.

[BL] has replaced BE as a supported, up-to-date cross-platform client tool... <whatever Pablo wants here>...

Matt, what are you thoughts on setting expectations clearly with something like the above?

d